csm/conf/configure.py

```diff
@@ -361,8 +361,20 @@ def _mem_limit_to_int(mem: str) -> int:
         :param mem: value from Conf as string.
         :returns: integer value.
         """
-        numbers_only = mem[:mem.find('M')]
-        return int(numbers_only)
+#         numbers_only = mem[:mem.find('M')]
+#         return int(numbers_only)
+        num = ''
+        unit = ''
+        for char in mem:
+            if ord(char) >= 48 and ord(char) <= 57:
+                num = num + char
+            elif (ord(char) >= 65 and ord(char) <= 90) or (ord(char) >= 97 and ord(char) <= 122):
+                unit += char
```

Comment:
As per this logic if  mem="12b3c5b", then result will be something like below,
num = 1235
unit = bcb

This is not the expected result. 

Try to use regex here to separate out mem value and mem unit.

Reply:
ok I'll add regex